### ISSUE TITLE
[BUG] Crash when refreshing tokens after added external ids

### ISSUE THREAD

**Issue by petyos** [rokwire/core-building-block#526]
**Describe the bug**
One we added:
```
"external_id_fields": {
        "uin": "uiucedu_uin",
        "net_id": "preferred_username"
 }
```
The service started crashing with:
`assignment to entry in nil map` error when the client hitting Refresh/Login API.

**Comment by shurwit** [comment]
Hi @petyos, thank you for reporting this! I see that you have self-assigned the issue, but let me know if you would like us to fix this instead.

**Comment by roberlander2** [comment]
Thanks @petyos, this bug should be fixed in #487. Please let me know if you think there is anything missing from those changes that is needed to address this issue.
